### DISCUSSION TITLE
Selection checkbox is not aligned vertically by default

### DISCUSSION THREAD

**Discussion by danielbettech** [icflorescu/mantine-datatable#136] [Q&A]
Hey!

I have noticed that checkbox is not aligned vertically to the middle of the row by default. Can we fix that?

A simple `display: flex;` to the checkbox component will make the job.

<img width="937" alt="image" src="https://user-images.githubusercontent.com/23063420/204435712-daeacf7c-2709-4d7f-9e73-a1220c54acaf.png">

**Comment by icflorescu** [comment]
That would conflict with cells `verticalAlignment`...
